script/os-autoinst-openvswitch

```diff
@@ -227,18 +229,15 @@ sub show ($self) {
     return $return_code, $ovs_vsctl_output;
 }
 
-################################################################################
-package main;
-use Mojo::Base -strict, -signatures;
+sub main () {
```

Comment:
I think it would still be good to have a separation, that means keeping the `OVS` package and the `main` package.

Reply:
And the `usage` and `GetOptions` part should be moved to the `main` package